pkg/apis/networking/types.go

```diff
@@ -22,10 +22,10 @@ import (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // AppliedToGroup is the message format of antrea/pkg/controller/types.AppliedToGroup in an API response.
 type AppliedToGroup struct {
-	metav1.TypeMeta
-	metav1.ObjectMeta
+	metav1.TypeMeta   `json:",inline"`
```

Comment:
internal unversioned type is not for external use and doesn't need json/yaml tag, please use versioned type instead.

Reply:
Removed.